ops/demo/alerts.tf

```diff
@@ -3,8 +3,14 @@ module "metric_alerts" {
   env                 = local.env
   app_service_plan_id = module.simple_report_api.app_service_plan_id
   app_service_id      = module.simple_report_api.app_service_id
-  action_group_id     = data.terraform_remote_state.global.outputs.slack_alert_action_id
   severity            = 1
   rg_name             = data.azurerm_resource_group.rg.name
   tags                = local.management_tags
+  mem_threshold       = 85
+
+  action_group_ids = [
+    data.terraform_remote_state.global.outputs.slack_alert_action_id,
```

Comment:
Relatedly, do we, in fact, want to page on every current alert? I don't want to be paged in the middle of the night because demo's memory usage is high. See larger question in main comments.

Reply:
I assume we don't want to page on non-prod events, but I was proposing to handle that by having the non-prod PagerDuty service not page people. (Which, in fact, is what it isn't doing right now.) It can just send a passive-aggressive emails instead.

Reply:
The way I configured the lower environments when I setup the PagerDuty services is they will only page during business hours, but notify to Slack at any time. We can customize the hours there or remove the paging completely.

Reply:
@jdorothy What do you think about sending all alerts to PagerDuty, and we handle how it alerts from there instead of in our Terraform? That way Terraform only is has the PagerDuty integration code.

Reply:
@ronaldheft-usds That did not occur to me but I love it. Fits perfectly with the general approach of keeping environments/infrastructure code uniform and externalizing the business logic.